modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-catalog-test/src/testIntegration/java/com/liferay/headless/commerce/delivery/catalog/resource/v1_0/test/ProductResourceTest.java

```diff
@@ -94,6 +94,29 @@ protected Long testGetChannelProduct_getChannelId() throws Exception {
 		return _commerceChannel.getCommerceChannelId();
 	}
 
+	@Override
+	protected Product testGetChannelProductByFriendlyUrlPath_addProduct()
+		throws Exception {
+
+		return _addCPDefinition(randomProduct());
+	}
+
+	@Override
+	protected Long testGetChannelProductByFriendlyUrlPath_getChannelId()
+		throws Exception {
+
+		return _commerceChannel.getCommerceChannelId();
+	}
+
+	@Override
+	protected String testGetChannelProductByFriendlyUrlPath_getFriendlyUrlPath()
+		throws Exception {
+
+		return _addCPDefinition(
+			randomProduct()
+		).getName();
+	}
+
```

Comment:
I don't think it's correct because this method should return the friendlyUrlPath of a product